<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QWidget" name="Find">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget_replace_find">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_find">
       <attribute name="title">
        <string>find</string>
       </attribute>
       <widget class="QLabel" name="label_find_target">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>find target</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_find_find">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>10</y>
          <width>160</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_find_next">
           <property name="text">
            <string>find next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_find_count">
           <property name="text">
            <string>count</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_find_cancel">
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_replace">
       <attribute name="title">
        <string>replace</string>
       </attribute>
       <widget class="QLabel" name="label_repalce_target">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>50</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>find target</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_repalce_target">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_replace_with">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>replace with</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_replace_with">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>100</y>
          <width>251</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>160</width>
          <height>261</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_replace_all_2">
           <property name="text">
            <string>find next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_replace_replace">
           <property name="text">
            <string>replace</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_replace_all">
           <property name="text">
            <string>replace all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_replace_cancel">
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_mark">
       <attribute name="title">
        <string>mark</string>
       </attribute>
       <widget class="QLabel" name="label_mark_target">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>find target</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mark_target">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>221</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>30</y>
          <width>160</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_mark_all">
           <property name="text">
            <string>mark all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_mark_clear">
           <property name="text">
            <string>clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_mark_cancel">
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
